docs/_includes/extra/script.js

```diff
@@ -2,6 +2,26 @@
 *  doesn't work due to fixed body height when menu is toggled.
 *  This script intercepts clicks on links, toggles the menu off
 *  and performs the anchor navigation. */
+
+window.addEventListener("load", function () {
+    let href = window.location.pathname;
+    const hash = window.location.hash;
+    if (hash!=="") {
+        href=hash
+    }
+    let element = document.querySelector(`a[href="${href}"]`);
+    if (element) {
+        element.scrollIntoView({ behavior: "smooth" });
+    } else {
+        href = window.location.pathname;
+        element = document.querySelector(`a[href="${href}"]`);
+        if (element) {
+            element.scrollIntoView({ behavior: "smooth" });
+        }
+    }
```

Comment:
```suggestion
    let element = document.querySelector(`a[href="${href}"]`);
    if (!element) {
         href = window.location.pathname;
        element = document.querySelector(`a[href="${href}"]`);
    } 
    element.scrollIntoView({ behavior: "smooth" });
```

should be simpler to read like this

Reply:
also worth to check if element isn't nil after second try

Reply:
logic updated. as for me made it better